src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -96,10 +96,22 @@ <h5>
                 </div>
                 <div class="d-flex">
                     <div class="form-group flex-grow-1">
-                        <jhi-date-time-picker labelName="{{ 'artemisApp.course.startDate' | artemisTranslate }}" formControlName="startDate"></jhi-date-time-picker>
+                        <jhi-date-time-picker
+                            labelName="{{ 'artemisApp.course.startDate' | artemisTranslate }}"
+                            formControlName="startDate"
+                            [(ngModel)]="course.startDate"
+                            [error]="!isValidStartDate"
+                        ></jhi-date-time-picker>
+                        <span *ngIf="!isValidEndDate" class="invalid-feedback"> {{ 'artemisApp.course.startDateInvalidWarning' | artemisTranslate }}</span>
                     </div>
                     <div class="form-group flex-grow-1 ms-3">
-                        <jhi-date-time-picker labelName="{{ 'artemisApp.course.endDate' | artemisTranslate }}" formControlName="endDate"></jhi-date-time-picker>
+                        <jhi-date-time-picker
+                            labelName="{{ 'artemisApp.course.endDate' | artemisTranslate }}"
+                            formControlName="endDate"
+                            [(ngModel)]="course.endDate"
+                            [error]="!isValidEndDate"
```

Comment:
Is there a reason why you did the check manually this way, and not use the native functionality?

You can add (custom) validators to the fields of the form in the ts file of the component, see ``course-update.component.ts``, lines 124 to 185, and [the Angular docs](https://angular.io/guide/form-validation#defining-custom-validators). Check the field for the short name to see how you can add the messages below the fields in case one of the validator functions fails

Reply:
![image](https://user-images.githubusercontent.com/60005702/169860967-5e677431-32b2-471b-8680-b4d47493e452.png)

mhm i guess i know why this wasn't done with the date pickers ^^

Reply:
Seems like a pure CSS issue, though :D
You could even keep your old style for the red hint